Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -252,7 +252,7 @@ DECLARE_SOA_COLUMN(Length, length, float);
 DECLARE_SOA_COLUMN(TOFExpMom, tofExpMom, float);                                              //! TOF expected momentum obtained in tracking, used to compute the expected times
 DECLARE_SOA_COLUMN(TrackEtaEMCAL, trackEtaEmcal, float);                                      //!
 DECLARE_SOA_COLUMN(TrackPhiEMCAL, trackPhiEmcal, float);                                      //!
-DECLARE_SOA_COLUMN(TrackTime, trackTime, float);                                              //! Estimated time of the track in ns wrt collision().bc() or ambiguoustrack.bcSlice()[0]
+DECLARE_SOA_COLUMN_FULL(TrackTimeImpl, trackTimeImpl, float, "trackTime");                    //! Implementation of estimated time of the track in ns wrt collision().bc() or ambiguoustrack.bcSlice()[0], use the corresponding getter
```

Comment:
Actually I overlooked yesterday. Shouldn't this abstract the timeErr and not the time itself? The time itself is unchanged in your proposal B...

Reply:
yes, of course. This was just to showcase and see if this abstraction actually works. I will change it now to something based on the other pr.